### PR TITLE
Add README detail

### PR DESCRIPTION
**Pull request by dk4g**
## What? 
Added details about the FMR service and the guide on how to run the application locally

## Why? 
So dev users can more easily install and run the app on their local system.

## Check list

- [ ] I have reviewed my own pull request
- [ ] I have written tests (if relevant)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,99 @@
-# evisa-find-my-reference
\ No newline at end of file
+# eVisa Find My Reference (FMR)
+
+eVisa Find My Reference (FMR) used by the public to request their reference number, which is required to access their eVisa.
+
+## Description
+
+- The service allows the user to make a request as the applicant themselve, or by someone who is helping the applicant, or by the applicant's legal representative.
```
Comment (jamiecarterHO) [inline README.md:7]: themselves needs an s on the end

File: README.md [hunk 2]
```diff
@@ -1 +1,99 @@
...
+
+By following these steps, you should be able to install and run your application using a Docker Compose. This provides a consistent development environment across different machines and ensures that all required dependencies are available.
+
+1. Make sure you have Docker installed and running on your machine. Docker is needed to create and manage your containers.
+
+2. To configure your dev environment, copy `/.devcontainer/devcontainer.env.sample` to `devcontainer.env` in the same directory and fill in the necessary values. This ensures your development container is set up with the required environment variables.
+
+3. Open a terminal, navigate to the project directory and run: `docker compose -f .devcontainer/docker-compose.dev.yml up -d`
+
+4. Once the containers are built and started, you can go inside the app container: `docker exec -it devcontainer-hof-coa-app-1 sh` (note: Docker containers may be named differently)
```
Comment (jamiecarterHO) [inline README.md:57]: hof-fmr-app in this case

File: README.md [hunk 3]
```diff
@@ -1 +1,99 @@
...
+7. Once the devcontainer is built and started, you will be inside the containerised environment. You can now work on your project as if you were working locally, but with all the necessary dependencies and tools installed within the container.
+
+8. To start the application, open a terminal within VS Code by going to `View -> Terminal` or by pressing `Ctrl+backtick` (`Cmd+backtick` on macOS). In the terminal, navigate to the project directory if you're not already there.
+
+9. Run the necessary commands to install dependencies `yarn` and `yarn start:dev` to start your application.
+
+### Testing
+
+#### Linting Tests
+`$ yarn run test:lint`
```
Comment (jamiecarterHO) [inline README.md:92]: can just be `yarn test:lint` but this doesn't matter

**Review by jamiecarterHO — COMMENTED** [review]
Small change where a coa was left in, couple of other tiny suggestions.

File: README.md (hunk 3 above)
Comment (dk4g) [reply README.md:92]: Updated. assume that users are familiar with Yarn shorthand commands

**Review by jamiecarterHO — APPROVED** [review]
lgtm

**Pull request merged**